base/deprecated.jl

```diff
@@ -936,15 +936,8 @@ end
 @deprecate super(T::DataType) supertype(T)
 
 function with_output_limit(thk, lim=true) # thk is usually show()
-    depwarn("with_output_limit is deprecated. use `io = IOContext(io, :limit_output => lim)` as a replacement", :with_output_limit)
-    global _limit_output
-    last = _limit_output
-    _limit_output::Bool = lim
-    try
-        thk()
-    finally
-        _limit_output = last
-    end
+    depwarn("with_output_limit is deprecated. use `io = IOContext(io, :limit => lim)` as a replacement", :with_output_limit)
+    thk()
```

Comment:
Coverage.jl had apparently been using it? https://github.com/JuliaLang/julia/pull/14534#issuecomment-169127647